hash/nes.xml

```diff
@@ -58194,12 +58199,13 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 		</part>
 	</software>
 
-	<software name="dquest5" cloneof="dwarr2">
-		<description>Yong Zhe Dou E Long V (China, decrypted)</description>
+<!-- Game has no sound. Does this freeze up? -->
+	<software name="dquest5" cloneof="dwarr2" supported="no">
+		<description>Yǒngzhě Dòu Èlóng V (China, decrypted)</description>
 		<year>19??</year>
 		<publisher>Waixing</publisher>
 		<info name="serial" value="ES-1106"/>
-		<info name="alt_title" value="勇者斗恶龙Ⅴ"/>
+		<info name="alt_title" value="勇者斗悪龍Ⅴ"/>
```

Comment:
Considering the use of traditional characters, was this actually intended for mainland China, or was it intended Taiwan/Hong Kong?  I’ve been wondering about a number of Waixing games that use traditional characters in the titles.  Waixing was based in Fuzhou (Fujian province), so the games were *developed* in mainland China, but do we have any indication of the intended market?

Reply:
Don't they use traditional characters on certain products to lend an air of gravitas in China? It's not unusual to see that in Japan. Anyway, my impression on the whole is that Waixing (and other) Chinese games tend to use simplified in-game but often traditional on the title, so I always assumed the (China) label was correct.